test/Microsoft.OpenApi.Tests/Validations/OpenApiSchemaValidationTests.cs

```diff
@@ -115,7 +115,7 @@ public void ValidateDefaultShouldNotHaveDataTypeMismatchForComplexSchema()
             var schema = new OpenApiSchema
             {
                 Type = JsonSchemaType.Object,
-                Properties =
+                Properties = new Dictionary<string, IOpenApiSchema>
```

Comment:
Does the shortcut syntax work where you only put "new" and the type is inferred?

Reply:
No that doesn't work. 
Properties is an implementation of the `IDictionary` interface https://github.com/microsoft/OpenAPI.NET/blob/5d99cc422c7646ec74528287003c6f449e421eb9/src/Microsoft.OpenApi/Models/OpenApiSchema.cs#L203  so you have to explicitly define the concrete type here.

Reply:
but we could change the property type as well. (see my latest comment)